### PR TITLE
💅 tests

### PR DESCRIPTION
**Pull request by folkyatina**
- updated prettier and eslint 
- auto format all the tests
- pump a pre-commit hook

A big ask to everyone for running `yarn && yarn husky install` after the merge

### CODE REVIEW

File: test/0.8.9/deposit-security-module.test.js [hunk 1]
```diff
@@ -819,10 +871,7 @@ contract('DepositSecurityModule', ([owner, stranger, guardian]) => {
         assert.isTrue((await depositSecurityModule.getGuardians()).includes(GUARDIAN1))
       })
       it(`setGuardianQuorum can't be called by non-admin`, async () => {
-        await assertRevert(
-          depositSecurityModule.setGuardianQuorum(1, { from: stranger }),
-          `NotAnOwner("${stranger}")`
-        )
+        await assertRevert(depositSecurityModule.setGuardianQuorum(1, { from: stranger }), `NotAnOwner("${stranger}")`)
```
Comment (skozin) [inline test/0.8.9/deposit-security-module.test.js:874]: I'd prefer it not reformatting `assert.revert`s like this but not sure we can configure it that way

File: test/0.8.9/oracle/hash-consensus-happy-path.test.js [hunk 2]
```diff
@@ -21,36 +37,36 @@ contract('HashConsensus', ([admin, member1, member2, member3, stranger]) => {
...
+      const deployed = await deployHashConsensus(admin, { initialEpoch: INITIAL_EPOCH })
       consensus = deployed.consensus
       reportProcessor = deployed.reportProcessor
     })
 
     it('adding members', async () => {
       await consensus.addMember(member1, 1, { from: admin })
       assert.equal(await consensus.getIsMember(member1), true)
-      assert.equal(+await consensus.getQuorum(), 1)
+      assert.equal(+(await consensus.getQuorum()), 1)
```
Comment (skozin) [inline test/0.8.9/oracle/hash-consensus-happy-path.test.js:48]: can we allow `+await`? the extra parentheses create a lot of visual noise

File: test/0.8.9/staking-router-keys-reporting.test.js [hunk 3]
```diff
@@ -482,31 +514,34 @@ contract('StakingRouter', ([deployer, lido, admin]) => {
         assert.equal(callInfo2.onExitedAndStuckValidatorsCountsUpdated.callCount, 0)
       })
 
-      it(`calling onValidatorsCountsByNodeOperatorReportingFinished doesn't call ` +
-         `anything on any module`, async () =>
-      {
-        await router.onValidatorsCountsByNodeOperatorReportingFinished({ from: admin })
+      it(
+        `calling onValidatorsCountsByNodeOperatorReportingFinished doesn't call ` + `anything on any module`,
```
Comment (skozin) [inline test/0.8.9/staking-router-keys-reporting.test.js:518]: don't need `+` here and in similar places anymore

Comment (skozin) [reply test/0.8.9/staking-router-keys-reporting.test.js:518]: also, the fact that a long test name idents it more than other tests at the same level but with shorter names significantly complicates visual navigation through the file, imo

File: test/0.8.9/oracle/hash-consensus-happy-path.test.js (hunk 2 above)
Comment (folkyatina) [reply test/0.8.9/oracle/hash-consensus-happy-path.test.js]: fixed

File: test/0.8.9/staking-router-keys-reporting.test.js (hunk 3 above)
Comment (folkyatina) [reply test/0.8.9/staking-router-keys-reporting.test.js]: Tried to fix it.

File: test/0.8.9/deposit-security-module.test.js (hunk 1 above)
Comment (folkyatina) [reply test/0.8.9/deposit-security-module.test.js]: Probably, we can't do anything here.

File: .eslintrc [hunk 4]
```diff
@@ -17,29 +17,26 @@
...
     "no-multiple-empty-lines": "warn",
-    "comma-dangle": "warn",
     "quotes": ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": true }],
     "jsx-quotes": ["error", "prefer-single"],
     "no-useless-escape": "off",
-    "linebreak-style": ["error", "unix"]
+    "linebreak-style": ["error", "unix"],
+    "no-restricted-modules": ["error", {
+        "name": "@aragon/contract-helpers-test/src/asserts",
+        "message": "Please use helpers/assert instead."
```
Comment (TheDZhon) [inline .eslintrc:32]: :+1: 
though, maybe it's better to make the fix more precise (specify path from the project's root)

File: test/0.4.24/stethpermit.test.js [hunk 5]
```diff
@@ -309,12 +287,21 @@ contract('StETHPermit', ([deployer, ...accounts]) => {
       // create a signed permit for a transfer
       const validAfter = 0
       const nonce = hexStringFromBuffer(crypto.randomBytes(32))
-      const { v, r, s } = signTransferAuthorization(from, to, value, validAfter, validBefore, nonce, domainSeparator, alice.key)
+      const { v, r, s } = signTransferAuthorization(
```
Comment (TheDZhon) [inline test/0.4.24/stethpermit.test.js:290]: ooph

**Review by TheDZhon — APPROVED** [review]
Like it :+1:

**Review by skozin — APPROVED** [review]

**Pull request merged**